spec/lib/twingly/http_spec.rb

```diff
@@ -692,6 +758,7 @@ class CustomError < StandardError; end
 
   describe "#put", vcr: Fixture.put_httpbin_org do
     include_examples "common HTTP behaviour for", :put, "https://httpbin.org/put"
+    include_examples "configures proxy correctly", :put
```

Comment:
We should add the `"verifies proxy functionality"` shared examples to the put/patch/delete tests as well:

```suggestion
    include_examples "verifies proxy functionality"
    include_examples "configures proxy correctly", :put
```

Reply:
Added in 7894f8d :)